### ISSUE TITLE
What does 'enable the reconstruct loss' do?

### ISSUE THREAD

**Issue by ruizewang** [Sampson-Lee/myDRGAN#2]
Hello Sampson,
I saw in train_single_DRGAN.py you add 
`                if args.use_rec:
                    g_loss += 10*L_g_rec`
Could you please tell me what it does for?

When I were training with enabling the reconstruct loss, after many epochs(about 50 epochs), the genbySimage seemed good. However, when I used the model to generate images,  the faces were very blurred.
![image](https://user-images.githubusercontent.com/8149167/39415654-8964ee6a-4c78-11e8-95b5-eff08067fb01.png)
![image](https://user-images.githubusercontent.com/8149167/39415640-5d77e992-4c78-11e8-8914-464de7858bc9.png)

**Comment by NomiMalik0207** [comment]
hello @22881916  i am trying to train it but in my case its taking too much time while training can you share me your training time??
